pkg/controller/volume/attachdetach/reconciler/reconciler.go

```diff
@@ -143,11 +143,20 @@ func (rc *reconciler) isMultiAttachForbidden(volumeSpec *volume.Spec) bool {
 			volumeSpec.Volume.Cinder != nil {
 			return true
 		}
+		// Check for volume types which do not fail when trying to multi-attach
+		if volumeSpec.Volume.VsphereVolume != nil {
```

Comment:
This is not necessary since the default behavior should `return false`. I don't want this to become a list of every plugin that exists.